frontend/src/lib/components/LemonTextArea/LemonTextArea.tsx

```diff
@@ -0,0 +1,95 @@
+import './LemonTextArea.scss'
+import React, { useRef, useState } from 'react'
+import { LemonInputPropsBase } from 'lib/components/LemonInput/LemonInput'
+import { LemonRow, LemonRowProps } from 'lib/components/LemonRow'
+import clsx from 'clsx'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconClose } from 'lib/components/icons'
+
+export interface LemonTextAreaProps
+    extends Omit<
+            React.TextareaHTMLAttributes<HTMLTextAreaElement>,
+            'value' | 'defaultValue' | 'onChange' | 'prefix' | 'suffix'
+        >,
+        LemonInputPropsBase {
```

Comment:
Abstraction nit: I'd say this probably shouldn't extend `LemonInputProps(Base)` since `LemonInput` is not actually used as part of `LemonTextArea` (for good reason). Hence the API of `LemonInput` will change independently from that of this component and affect `LemonTextArea` when that _shouldn't_ be the case (i.e. `LemonInput` might support numbers, but `LemonTextArea` never will).